src/components/error-message/error-message.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+@Component({
+  selector: 'error-message',
+  templateUrl: 'error-message.html',
+})
+export class ErrorMessageComponent {
+
+  @Input() returnTo: string;
+  @Input() showAdditionalText: boolean;
+
+  @Output() exitModal = new EventEmitter<string>();
```

Comment:
Could be `EventEmitter<void>` if you never call `emit` with any argument.

Reply:
Updated in latest push.